lib/manageiq/api/common/graphql/generator.rb

```diff
@@ -67,6 +67,16 @@ def self.collection_field_resolvers(schema_overlay, collection)
             field_resolvers
           end
 
+          def self.collection_schema_overlay(schema_overlay, collection)
+            collection_schema_overlay = {}
+            schema_overlay.keys.each do |collection_regex|
+              next unless collection.match(collection_regex)
```

Comment:
Is this an actual regex here or a string?

Reply:
Also, since you are not using the return value of the match, nor any magic like `$1`, prefer `.match?` here

Reply:
these are regex'es